lib/widgets/modular_widgets/wallet_connect_widgets/wallet_connect_pairing_list_card.dart

```diff
@@ -0,0 +1,153 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:walletconnect_flutter_v2/walletconnect_flutter_v2.dart';
+import 'package:zenon_syrius_wallet_flutter/blocs/blocs.dart';
+import 'package:zenon_syrius_wallet_flutter/blocs/wallet_connect/wallet_connect_pairings_bloc.dart';
+import 'package:zenon_syrius_wallet_flutter/blocs/wallet_connect/wallet_connect_sessions_bloc.dart';
+import 'package:zenon_syrius_wallet_flutter/main.dart';
+import 'package:zenon_syrius_wallet_flutter/services/wallet_connect_service.dart';
+import 'package:zenon_syrius_wallet_flutter/utils/extensions.dart';
+import 'package:zenon_syrius_wallet_flutter/widgets/reusable_widgets/icons/clear_icon.dart';
+import 'package:zenon_syrius_wallet_flutter/widgets/widgets.dart';
+
+const String _kWidgetTitle = 'WalletConnect Pairing List';
+const String _kWidgetDescription =
+    'A pairing refers to the initial process of establishing a secure connection between a wallet and the dApp.'
+    'Once the pairing is complete, a session is established, which allows the dApp to communicate with the wallet '
+    'securely over the Internet';
+
+class WalletConnectPairingsCard extends StatefulWidget {
+  const WalletConnectPairingsCard({Key? key}) : super(key: key);
+
+  @override
+  State<WalletConnectPairingsCard> createState() =>
+      _WalletConnectPairingsCardState();
+}
+
+class _WalletConnectPairingsCardState extends State<WalletConnectPairingsCard> {
+  final WalletConnectPairingsBloc _pairingsBloc =
+      sl.get<WalletConnectPairingsBloc>();
+
+  @override
+  void initState() {
+    // Initialize WalletConnect client
+    sl.get<WalletConnectService>().initClient();
```

Comment:
`WalletConnectService` should only be registered once (in `InitUtils`), otherwise the event handlers are registered multiple times causing unexpected behavior. Maybe a boolean should be added to `WalletConnectService` to prevent it from being initialized twice accidentally.